### DISCUSSION TITLE
widgets and dashboard tabs not available

### DISCUSSION THREAD

**Discussion by mehdi602** [Serial-Studio/Serial-Studio#69] [General]
hello, I have the widgets and dashboard tabs not available, is this normal ? 
Thank you for the project.

**Comment by alex-spataru** [comment]
Hi! This is the normal behavior of Serial Studio when it cannot parse the data received by the microcontroller. This can be for the following reasons:
- If in *auto mode*, the JSON string sent by the MCU is incorrect.
- If in *manual mode*, the data of the microcontroller cannot be mapped with the JSON file.
- You are not sending the start/end sequence characters (`/*`and `*/`, you can change these strings from in the `Setup` → `Settings` tab).

If possible, please share a screenshot of Serial Studio with the console output and your JSON file so that I can help you sort this problem out.

**Comment by alex-spataru** [comment]
@mehdi602 the [latest](https://github.com/Serial-Studio/Serial-Studio/releases/tag/v1.0.22) release features a JSON editor, which allows you to create, edit & validate JSON files with a graphical user interface (instead of manually generating your JSON file). Please let me know if it helps you with your issue.

**Reply by mehdi602** [reply]
Thanks @alex-spataru for the help it works perfectly now. And it's a good idea the json editor, very useful.
Surely I had an error in my json file.
